api/routes/middlewares/index.js

```diff
@@ -50,6 +50,12 @@ import passport from 'passport';
 middlewares.use(passport.initialize());
 middlewares.use(passport.session());
 
+// rate limiter middleware to reduce API calls to server within time frame
+import rateLimiter from 'shared/middlewares/rateLimiter';
+const THREE_HUNDRED = 300;
```

Comment:
Hello @LetItRock,

What do you think about giving this a more _generic_ name, maybe something like `BOTTOM_LIMIT` and `TOP_LIMIT`? Imagine if you Spectrum decided to change `300` to `800`, then you'd have to change the variable name too.

Reply:
That's a good point; could be like:
`REQUESTS_ALLOWED` and `TIMEFRAME` - generic enough, and maps to what the arguments are doing in the function?

Reply:
sure I'll do that, you are both right its better to have here more generic name...